index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Concert</title>
+    <link rel="stylesheet" href="./style.css" />
+    <link rel="stylesheet" href="https://cdn.lineicons.com/4.0/lineicons.css" />
+</head>
+<body>
+    <div class="top">
+        <ul>
+            <li><i class="lni lni-facebook-fill"></i></li>
+            <li><i class="lni lni-twitter-original"></i></li>
+            <li><span>Logout</span></li>
+            <li><span>Mypage</span></li>
+            <li><span>Korean</span></li>
+        </ul>
+    </div> 
+    <div class="main-container">
+        <div class="top-bar">
+            <nav>
+                <img class="close-menu" src="./images/x-solid.svg" alt="Close menu">
+                <div class="hamburger">
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                </div>
+                <div class="menu-img-holder">
+                    <a href="./index.html"><img class="menu-logo" src="./images/logo.png" alt=""></a>
+                </div>
+                <div class="menu">
+                    <ul>
+                        <li><a onclick="location.href='about.html'">About</a></li>
```

Comment:
- [Optional ] You did a great job, however, I recommend using `cursor: pointer`.